### ISSUE TITLE
Add label on "Choose File" button in Upload to ScholarSphere section

### ISSUE THREAD

**Issue by anaelizabethenriquez** [psu-libraries/researcher-metadata#607]
On Publication Details page in “Upload to ScholarSphere” box, the file upload box ("Choose File") needs a label. In user testing, the screen reader read "Button Required Invalid Entry” on that item. There should be a label saying “Choose File.” 

For a link to the recording of this user testing session, see [Notes from October 2022 RMD user testing](https://pennstateoffice365.sharepoint.com/:w:/r/sites/OpenAccessPolicyInfrastructure/_layouts/15/Doc.aspx?sourcedoc=%7Bbd1516e6-2d75-433f-877c-cb463d8a2035%7D&action=editnew&cid=8796e4ea-dc04-c4ae-5793-16cd16cd2ec9) (section header is Michelle McManus, October 26, 2022). This issue came up at about the 10-minute mark.
